askama_derive/src/input.rs

```diff
@@ -167,6 +163,17 @@ impl From<String> for EscapeMode {
     }
 }
 
+impl<'a> From<&'a PathBuf> for EscapeMode {
```

Comment:
I don't think using `From` here is idiomatic, it is not really a type conversion. I think it might make sense to add a `from_path()` to an `impl` block for `EscapeMode`, though.